templates/zerver/help/contact-support.md

```diff
@@ -17,15 +17,15 @@ channels:
   app](https://github.com/zulip/zulip-mobile/issues/new), [desktop
   app](https://github.com/zulip/zulip-desktop/issues/new). If you
   aren't able to provide a clear bug report or are otherwise
-  uncertain, it can be helpful to discuss in chat.zulip.org first to
-  help create a better GitHub issue.
+  uncertain, it can be helpful to discuss in
+  [the Zulip developer community](https://zulip.com/developer-community/) first to help create a better GitHub issue.
 * We don't offer phone support except as part of enterprise contracts.
 
 We love hearing feedback! Please reach out if you have feedback,
 questions, or just want to brainstorm how to make Zulip work for your
 organization.
 
-[chat-zulip-org]: /developer-community/
+[The Zulip developer community]: https://zulip.com/developer-community/
```

Comment:
I am not sure if this will work since there are spaces in between the words. Better to use something like `developer-community` for the named link.